helm/camel-k/README.md

```diff
@@ -56,15 +56,30 @@ additional parameters that can be set during installation.
 
 > **Tip**: List all releases using `helm list`
 
+## Upgrading the Chart
+
+If you are upgrading the `camel-k` Deployment, you should always use a specific version of the chart and pre-install the CRDS:
+
+```
+$ kubectl replace -k "https://github.com/apache/camel-k/config/crd?ref=vx.y.z"
```

Comment:
We should use the CRDs contained into the chart for consistency.

Reply:
I am not sure of this. The charts also contains the edit cluster role, so it would replacing them be one by one. 
AFAIK the CRDs in the charts should always be identical to the ones in config for one version. Also I wonder if we should improve the sync between both versions if we want consistency, WDYT?